typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -180,27 +129,64 @@ export const ChainMetadataSchemaObject = z.object({
     })
     .optional()
     .describe('Block settings for the chain/deployment.'),
-  transactionOverrides: z
-    .record(z.any())
+
+  chainId: z
+    .union([ZNzUint, z.string()])
+    .describe(`The chainId of the chain. Uses EIP-155 for EVM chains`),
+
+  customGrpcUrls: z
+    .string()
     .optional()
-    .describe('Properties to include when forming transaction requests.'),
+    .describe(
+      'Specify a comma separated list of custom GRPC URLs to use for this chain. If not specified, the default GRPC urls will be used.',
+    ),
+
+  deployer: z
+    .object({
+      name: z.string().describe('The name of the deployer.'),
+      email: z
+        .string()
+        .email()
+        .optional()
+        .describe('The email address of the deployer.'),
+      url: z.string().url().optional().describe('The URL of the deployer.'),
+    })
+    .optional()
+    .describe(
+      'Identity information of the deployer of a Hyperlane instance to this chain',
+    ),
```

Comment:
This is the only addition. The rest is just an alphabetical re-ordering of the fields